### PR TITLE
Fix/revpi 2856/adapt telegrams to serdev

### PR DESCRIPTION
**Pull request by linosanfilippo-kunbus**

### CODE REVIEW

**Comment by l1k** [comment]
Nit: Commit message of 9752968efb32544ac7156b49452ad7e25a4e9fb7 cites `pibridge_recv_timeout()` instead of `pibridge_req_io()`.

File: drivers/tty/serdev/pibridge.c [hunk 1]
```diff
@@ -193,12 +192,7 @@ int pibridge_recv_timeout(void *buf, u8 len, u16 timeout)
 
 	trace_pibridge_receive_end(buf, received);
 
-	if (received != len) {
-		trace_pibridge_receive_timeout(received, len, timeout);
```
Comment (l1k) [inline drivers/tty/serdev/pibridge.c:197]: Are you sure that you want to eliminate the tracepoint? I suspect this is erroneous, but if you do want to get rid of the tracepoint, you'd have to update the tracepoint documentation as well.

**Review by l1k — CHANGES_REQUESTED** [review]
Apart from the two small issues I've commented on, this LGTM.

File: drivers/tty/serdev/pibridge.c (hunk 1 above)
Comment (linosanfilippo-kunbus) [reply drivers/tty/serdev/pibridge.c:197]: Right, this tracepoint should stay. I will fix it.

**Review by l1k — APPROVED** [review]

**Pull request merged**